package.json

```diff
@@ -1,34 +1,27 @@
 {
   "description": "FutureDays Farm",
-  "engines": {
-    "node": "9.2.1"
-  },
   "dependencies": {
-    "bcrypt": "^1.0.2",
-    "email": "^0.2.6",
-    "googleapis": "^22.2.0",
+    "@conduitvc/appsync-emulator-serverless": "0.6.4",
+    "aws-appsync": "ConduitVC/aws-appsync",
+    "babel-core": "^6.26.3",
```

Comment:
dev dependency

Reply:
How can I tell if a module should be a devDependency, or just a dependency?

Reply:
devDependencies are things only needed for local development.  Because babel takes our code, and turns it into "compiled" code, its only needed for dev.

A library used in production, like `uuid` is a dependency.

Separating them allows sophisticated systems to only grab the smallest amount of code needed in production environments.  Making things as small, and fast as possible